### PR TITLE
Add statement_timeout to PostgreSQL connection parametres.

### LINKED ISSUE #129: Add timeout support for postgresql
this is just not to forget...

we can implement timeouts in postgresql backend by issuing the `SET SESSION statement_timeout TO NNN` command right after opening a connection to database. `NNN` is in milliseconds.

When getting corresponding error back, we can wrap it into separate exception.

P.S. I think, that this is possible to implement timeouts not only for postgesql backend....

### PR DESCRIPTION
**Pull request by mloskot**
Closes #129

### CODE REVIEW

File: src/backends/postgresql/session.cpp [hunk 1]
```diff
@@ -74,6 +76,13 @@ void postgresql_session_backend::connect(
                          : "SET extra_float_digits = 2",
         "Cannot set extra_float_digits parameter");
 
+    {
+        std::ostringstream oss;
+        oss << "SET staetment_timeout = " << statement_timeout_;
```
Comment (Mingun) [inline src/backends/postgresql/session.cpp]: Misprint: `staetment_timeout` instead of `statement_timeout` (and also in line 83)

Comment (mloskot) [reply src/backends/postgresql/session.cpp]: @Mingun Thank you for the review. AT, Having no local dev environment for SOCI, I'm completely relying on the CI builds. I'll fix it.

File: docs/backends/postgresql.md [hunk 2]
```diff
@@ -51,7 +51,8 @@ The set of parameters used in the connection string for PostgreSQL is the same a
 
 In addition to standard PostgreSQL connection parameters, the following can be set:
 
-* `singlerow` or `singlerows`
+* `singlerow` or `singlerows` - to control single-row mode
+* `statement_timeout` - to set override statement default timeout (milliseconds)
```
Comment (vadz) [inline docs/backends/postgresql.md]: Either "set" or "override" seems to be superfluous here.

**Comment by mloskot** [comment]
@vadz `sql << "SET statement_timeout = whatever";` makes perfect sense. I agree, it's hard to justify the code clutter when one can achieve the goal via the existing SQL interface. 

I'm dropping the idea.

**Pull request closed without merging**